addon/mixins/validatable-input.js

```diff
@@ -165,7 +165,7 @@ export default Ember.Mixin.create({
     } else {
       parent.addClass('has-error');
       element.next('.input-error').remove();
-      element.after('<p class="input-error" role="alert">' + errorMessage + '</p>');
+      element.after(`<label class="input-error" role="alert" for="${element.attr("id")}">${errorMessage}</label>`);
```

Comment:
@bakura10, yes.  And since all error messages are for input controls, the ``label`` is the safest, most accessible choice.  The ``label`` is better than any other element because is automatically focuses the corresponding ``input``, ``select``, or ``textarea`` element when clicked.